### ISSUE TITLE
High CPU usage after few autocomplete suggestions 

### ISSUE THREAD

**Issue by msoedov** [codota/tabnine-vscode#56]
Reproducible 100% on different projects and files 
```shell
.vscode/extensions/tabnine.tabnine-vscode-2.1.11/binaries/2.1.17/x86_64-apple-darwin/TabNine -v
TabNine 2.1.17 (x86_64-apple-darwin)
Jacob Jackson <jacob@tabnine.com>
```

<img width="573" alt="Screen Shot 2019-10-13 at 15 54 26" src="https://user-images.githubusercontent.com/1958116/66715940-c9236a80-edd1-11e9-8061-861660454510.png">

**Comment by drorweiss** [comment]
We have a fix in beta (version 2.3.4) for excessive CPU and memory consumption by TabNine.
We'd appreciate it if you try it out and let us know if it solves the issue for you.
To try the fix, you have to enable the beta version. Just open TabNine's config by typing TabNine::config in your editor, and then check both **Update TabNine automatically** and **Receive beta releases of TabNine** Then, restart your editor, and press TabNine::confiig to verify you have TabNine 2.3.4. or newer.  Please let us know if this version solves the problem for you. We plan to make it available to all users in a couple of weeks.

**Comment by amircodota** [comment]
Hi @msoedov 

Any chance you can try the latest tabnine version? We've made several fixes to improve CPU and memory consumption, and I want to verify that ths issue you had in your use case is indeed fixed.
